test/regression_tests/thermal_bubble_regression_test.jl

```diff
@@ -11,7 +11,7 @@ function run_thermal_bubble_regression_test(arch, grid_type)
     end
 
     closure = ScalarDiffusivity(ν=4e-2, κ=4e-2)
-    model = NonhydrostaticModel(grid=grid, closure=closure, coriolis=FPlane(f=1e-4),
+    model = NonhydrostaticModel(; grid, closure, coriolis=FPlane(f=1e-4),
```

Comment:
Same comment as before. In my opinion `grid=grid, closure=closure` is more user friendly.

Reply:
hm... perhaps you have a point here